scripts/infra/setup-aws.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env -S bash -e
+# SPDX-License-Identifier: Apache-2.0
+
+ensure_system_pkgs() {
```

Comment:
Would this be better to create as a script that can have multiple sub commands.  This method for example should probably have rh in the name to leave room for other os types.

Reply:
Yes I think that does make sense